pep_sphinx_extensions/pep_processor/transforms/pep_headers.py

```diff
@@ -223,3 +254,47 @@ def _process_pretty_url(url: str) -> tuple[str, str]:
 def _make_link_pretty(url: str) -> str:
     item_name, item_type = _process_pretty_url(url)
     return f"{item_name} {item_type}"
+
+
+def _abbreviate_status(status: str) -> str:
+    if status in SPECIAL_STATUSES:
+        status = SPECIAL_STATUSES[status]
+
+    if status == STATUS_DRAFT:
+        return "Proposal under active discussion and revision"
+    if status == STATUS_DEFERRED:
+        return "Inactive draft that may be taken up again at a later time"
+    if status == STATUS_ACCEPTED:
+        return "Normative proposal accepted for implementation"
+    if status == STATUS_ACTIVE:
+        return "Currently valid/in-use guidance or process"
+    if status == STATUS_FINAL:
+        return "Accepted and implementation complete, or no longer active"
+    if status == STATUS_WITHDRAWN:
+        return "Removed from consideration by author(s)/sponsor"
+    if status == STATUS_REJECTED:
+        return "Formally declined and will not be accepted"
+    if status == STATUS_SUPERSEDED:
+        return "Replaced by another succeeding PEP"
+    if status == STATUS_PROVISIONAL:
+        return "Provisionally accepted but additional feedback needed"
```

Comment:
Wouldn't it be nicer and more Python to make this a dictionary lookup table (e.g. in a module-level constant) rather than a long chain of if statements?

Reply:
Yep, refactored!